app/routes/auth.tsx

```
@@ -0,0 +1,55 @@
+
+  return (
+    <main className="bg-[url('/images/bg-auth.svg')] bg-cover min-h-screen flex items-center justify-center"> 
+        <div className="gradient-border shadow-lg">
+            <section className="flex flex-col gap-8 bg-white rounded-2xl p-10">
+                <div className="flex flex-col items-center gap-2 text-center">
+                    <h1>Welcome</h1>
+                    <h2>Log In to Continue Your Job Journey</h2>
+                </div>
+                <div>
+                    {isLoading ? (
+                        <button className="auth-button animate-pulse">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)